pkg/scheduler/framework/plugins/noderesources/resource_allocation.go

```diff
@@ -66,10 +66,9 @@ func (r *resourceAllocationScorer) score(
 	score := r.scorer(requested, allocatable)
 
 	if klog.V(10).Enabled() {
-		klog.Infof(
-			"%v -> %v: %v, map of allocatable resources %v, map of requested resources %v ,score %d,",
-			pod.Name, node.Name, r.Name,
-			allocatable, requested, score,
+		klog.InfoS("Listing internal info for allocatable resources, requested resources and score", "pod",
+			klog.KObj(pod), "node", klog.KObj(node), "resourceAllocationScorer", r.Name,
+			"allocatable", allocatable, "requested", requested, "score", score,
```

Comment:
Does it make sense to add "resource" to these names to make them less generic. 
allocatable -> allocatableResource 
requested -> requestedResource
score -> resourceScore

Reply:
SGTM but I don't have any strong opinion on this
Pinging @serathius for suggestions on both comments

Reply:
I agree with @pchan suggestion. From log consumer that doesn't know the code `allocatable` doesn't provide enough information (same in other cases). We need to add information that logged value is a resource so users will properly interpret it.

Reply:
Done, thanks pchan!!